itou/templates/utils/templatetags/buttons_form.html

```diff
@@ -58,33 +61,18 @@
 <div id="confirm_reset_modal" class="modal fade" tabindex="-1" aria-hidden="true">
     <div class="modal-dialog modal-dialog-centered">
         <div class="modal-content">
-            {% if modal_content_save_and_quit %}
-                <div class="modal-header">
-                    <h3 class="modal-title">Enregistrer et quitter ?</h3>
-                </div>
-                <div class="modal-body">
-                    Les informations renseignées seront enregistrées.
-                    <br>
-                    Vous pourrez terminer de les compléter plus tard.
-                </div>
-                <div class="modal-footer">
-                    <button type="button" class="btn btn-sm btn-outline-primary" data-bs-dismiss="modal">Retour</button>
-                    <a href="{{ reset_url }}" class="btn btn-sm btn-danger">Enregistrer et quitter</a>
-                </div>
-            {% else %}
-                <div class="modal-header">
-                    <h3 class="modal-title">Êtes-vous sûr de vouloir annuler ?</h3>
-                </div>
-                <div class="modal-body">
-                    Les informations renseignées ne seront pas enregistrées.
-                    <br>
-                    Cette action est irréversible.
-                </div>
-                <div class="modal-footer">
-                    <button type="button" class="btn btn-sm btn-outline-primary" data-bs-dismiss="modal">Retour</button>
-                    <a href="{{ reset_url }}" class="btn btn-sm btn-danger">Confirmer l'annulation</a>
-                </div>
-            {% endif %}
+            <div class="modal-header">
```

Comment:
peut être aussi le mettre dans un `if secondary_name and secondary_value or secondary_url` ?